<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Nosto_Credit</class>
 <widget class="QDialog" name="Nosto_Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn40_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
   </property>
   <property name="text">
    <string>40</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn100_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>170</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Muu summa:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn500_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn200_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
   </property>
   <property name="text">
    <string>200</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNosta_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
   </property>
   <property name="text">
    <string>Nosta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn20_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>172</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn60_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
   </property>
   <property name="text">
    <string>60</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMenu_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>176</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣QPushButton {

  background-color:  #C0C0C0;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣

QPushButton:pressed {

  background-color:  #808080;
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: beige;
    min-width: 10em;
    padding: 6px;
	color: black;
}♣</string>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
